packages/components/src/Icon/Icon.component.js

```diff
@@ -63,17 +92,41 @@ function Icon({ className, name, title, transform, onClick, ...props }) {
 		className,
 		SVG_TRANSFORMS[transform],
 	);
-	const iconElement = (
-		<svg name={name} className={classname} {...accessibility} {...props}>
-			<use xlinkHref={IconsProvider.getIconHREF(name)} />
+	let iconElement = (
+		<svg
+			name={name}
+			className={classname}
+			ref={ref => {
+				if (!ref) {
+					return;
+				}
+				if (isRemote && content && content.includes('svg')) {
+					// eslint-disable-next-line no-param-reassign
+					ref.innerHTML = content;
+				}
+			}}
```

Comment:
Please use `useRef` and `useEffect` to set things in the ref if it is initialized